hbase/src/main/scala/akka/stream/alpakka/hbase/HBaseSettings.scala

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.hbase
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.hbase.TableName
+import org.apache.hadoop.hbase.client.Put
+
+final case class HTableSettings[T](conf: Configuration,
+                                   tableName: TableName,
+                                   columnFamilies: Seq[String],
+                                   converter: T => Put)
```

Comment:
`HTableSettings` is used as method parameter in `javadsl` and the scala function is not java friendly in Scala 2.11

I suggest that you add a java friendly factory method in the companion of `HTableSettings`, or use two different `scaladsl.HTableSettings` and `javadsl.HTableSettings`